zqd/search/search.go

```diff
@@ -37,9 +38,13 @@ func NewSearch(ctx context.Context, s *space.Space, req api.SearchRequest) (*Sea
 	if req.Span.Dur < 0 {
 		return nil, errors.New("time span must have non-negative duration")
 	}
-	// XXX allow either direction even through we do forward only right now
-	if req.Dir != 1 && req.Dir != -1 {
-		return nil, errors.New("time direction must be 1 or -1")
+	// XXX zqd only supports backwards searches, remove once this has been
+	// fixed.
+	if req.Dir == 1 {
+		return nil, zqe.E("forward facing searches not supported", zqe.Invalid)
```

Comment:
When giving arguments to `zqe.E`, a string (and any formatting verbs) need to be last, so that it can mimic fmt.Errorf's variadiac arguments.

```suggestion
		return nil, zqe.E(zqe.Invalid, "forward facing searches not supported")
```